packages/manager/apps/telecom/src/app/telecom/telephony/billingAccount/dashboard/dashboard.controller.js

```diff
@@ -264,6 +270,10 @@ export default /* @ngInject */ function TelecomTelephonyBillingAccountDashboardC
       });
   }
 
+  this.isGroupSuspended = function isGroupSuspended() {
```

Comment:
some nitpicking here but why do we have the ```isSuspended``` method that returns if a group is suspended and then we have the method ```isGroupSuspended```?

it looks like ```isGroupSuspended```is doing a bit more than just checking is a group is suspended, maybe rename it to something like ```shouldIncreaseDeposit``` or whatever makes sense in this context